src/utils/Processing.ts

```diff
@@ -99,7 +99,22 @@ export function loadExternalDependencies(
 }
 
 function getLakeOfFHIR(inDir: string): LakeOfFHIR {
-  const files = getFilesRecursive(inDir).filter(file => file.endsWith('.json'));
+  // Don't process files in .git folder
+  // Don't process any files in a temp folder, unless the temp folder is used in inDir when running GoFSH
+  const tempDir = `${path.sep}temp${path.sep}`;
+  const gitDir = `${path.sep}.git${path.sep}`;
+  const tempInInDir = inDir.endsWith(`${path.sep}temp`) || inDir.includes(tempDir);
+  const relevantFiles = getFilesRecursive(inDir).filter(
+    file => (tempInInDir ? true : !file.includes(tempDir)) && !file.includes(gitDir)
+  );
+
+  // IG Publisher creates .escaped.json files that are not valid JSON
+  const files = relevantFiles.filter(
+    file =>
+      file.endsWith('.json') &&
+      !file.endsWith('.escaped.json') &&
+      !IGNORED_RESOURCE_LIKE_FILES.some(path => file.endsWith(path))
+  );
```

Comment:
We debug log files we skip in line 124-128 below (and in my suggested code above).  I'm wondering, if, for consistency, we should also debug log the `.escaped.json` and `output/expansions.json` files too...  If so, you might want to put those checks in the for-loop below instead (before you read the file in), so you can debug log and early-return when you find them...

Reply:
This would also allow your tests to be a little more explicit by actively checking for the debug log statement instead of just testing the number of things returned.

Reply:
Actually... you could just pop these into `getFilesRecursive` too (since they're only based on file name/path).

Reply:
I hadn't added a debug statement for these files because they felt like a slightly different category to me. These seemed like files we definitely didn't want to process, just like files that are other file types that we automatically filter out. However, maybe that's too much of a distinction. A debug statement won't hurt anyone. I can add them into the `getFilesRecursive`, as long as you agree that these are files we'd want to skip any time we try to get a list of files.